entities.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from abc import ABC, abstractmethod
+
+
+class Position(object):
+
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
+    def move(self, x, y):
+        self.x += x
+        self.y += y
+
+    def __repr__(self):
+        return f"({self.x}, {self.y})"
+
+
+STARTING_PLAYER_POS = Position(0, 0)
```

Comment:
The dungeon generation algorithm should take care of the starting position by placing the player (or the stairs) in a random room